### PR TITLE
blog: add native mobile auth blog article

### PR DESCRIPTION
**Pull request by dshukertjr**
## What kind of change does this PR introduce?

Adds native mobile auth blog article

### CODE REVIEW

File: apps/www/_blog/2023-06-26-native-mobile-auth.mdx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+![Supabase dashboard Google provider](/images/blog/native-mobile-auth/google-setting.png)
+
+It really is as simple as that.
+
+### Using Sign in with Apple in a Flutter App
+
+We introduced support for native Sign in with Apple in our [previous launch week](https://supabase.com/blog/supabase-auth-sso-pkce#native-apple-login-on-ios#native-apple-login-on-ios), but today we are adding support for multiple apps on a single Supabase project.
+
+All you need to do is to register all of the bundle IDs of your apps in the [Supabase Dashboard](https://supabase.com/dashboard/project/_/auth/providers) under \***\*\*\*\*\***\*\***\*\*\*\*\***Authorized Client IDs\***\*\*\*\*\***\*\***\*\*\*\*\*** as a comma separated string.
```
Comment (hf) [inline apps/www/_blog/2023-06-26-native-mobile-auth.mdx:42]:
```suggestion
All you need to do is to register all of the bundle IDs of your apps in the [Supabase Dashboard](https://supabase.com/dashboard/project/_/auth/providers) under _Authorized Client IDs_ as a comma separated string.
```

File: apps/www/_blog/2023-06-26-native-mobile-auth.mdx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+```tsx
+async function handleSignInWithGoogle(response) {
+  const { data, error } = await supabase.auth.signInWithIdToken({
+    token: response.credential,
+    nonce: 'NONCE', // must be the same one as provided in data-nonce (if any)
+  })
+}
+```
+
+## ## Resources
```
Comment (dshukertjr) [inline apps/www/_blog/2023-06-26-native-mobile-auth.mdx:130]:
```suggestion
## Resources
```

File: apps/www/_blog/2023-06-26-native-mobile-auth.mdx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  const { data, error } = await supabase.auth.signInWithIdToken({
+    token: response.credential,
+    nonce: 'NONCE', // must be the same one as provided in data-nonce (if any)
+  })
+}
+```
+
+## Resources
+
+Link to the new docs from this PR: https://github.com/supabase/supabase/pull/14802
```
Comment (dshukertjr) [inline apps/www/_blog/2023-06-26-native-mobile-auth.mdx:132]:
```suggestion
- [Login with Apple Guide](https://supabase.com/docs/guides/auth/social-login/auth-apple)
- [Login with Google Guide](https://supabase.com/docs/guides/auth/social-login/auth-google)
```

File: apps/www/_blog/2023-06-26-native-mobile-auth.mdx [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+    token: response.credential,
+    nonce: 'NONCE', // must be the same one as provided in data-nonce (if any)
+  })
+}
+```
+
+## Resources
+
+- [Login with Apple Guide](https://supabase.com/docs/guides/auth/social-login/auth-apple)
+- [Login with Google Guide](https://supabase.com/docs/guides/auth/social-login/auth-google)
```
Comment (dshukertjr) [inline apps/www/_blog/2023-06-26-native-mobile-auth.mdx:133]: @hf Feel free to add any other resources that you may want to include!

**Review by RamiroND — APPROVED** [review]

**Pull request merged**